specs/darwin/quarantine_history.table

```diff
@@ -0,0 +1,20 @@
+table_name("quarantine_history")
+description("OS X Quarantine Events (downloaded/streamed files).")
+schema([
+    Column("uid", BIGINT, "The user associated with the quarantine event", additional=True),
+    Column("id", TEXT, "Quarantine event ID (matches ID in quarantined file's extended attributes", index=True),
+    Column("agent_name", TEXT, "The name of the quarantining agent (application or program)"),
+    Column("agent_bundle_identifier", TEXT, "The bundle identifier of the quarantining agent, if available"),
+    Column("type", INTEGER, "A symbolic string identifying the why the item is quarantined, if available. 0 = Web Download, 1 = Other Download, 2 = Email Attachement, 3 = Instant Message Attachment, 4 = Calendar Event Attachment, 5 = Other Attachment, 6 = Airdrop Attachment"),
```

Comment:
What do you think of putting the string type in, would that be more useful than an integer whose meaning must be looked up?